insights/parsers/mongod_conf.py

```diff
@@ -0,0 +1,93 @@
+"""
+mongodb-server configuration files
+==================================
+
+This module contains the following files:
+    ``/etc/mongod.conf`` and ``/etc/mongodb.conf``.
```

Comment:
Missed the `etc/opt/rh/rh-mongodb26/mongod.conf`?

Reply:
Yes.